extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -39,6 +40,8 @@ export class AmpAdXOriginIframeHandler {
    * @param {!./amp-ad-3p-impl.AmpAd3PImpl|!../../amp-a4a/0.1/amp-a4a.AmpA4A} baseInstance
    */
   constructor(baseInstance) {
+    /** {!Window}*/
+    this.win = window;
```

Comment:
@cramforce But as far as I can see this class has no superclass.
It seems that the `window` property will be always available via `this.baseInstance_.win`.